tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+import base64
+import json
+import os
+
+import pytest
+import requests
+
+
+@pytest.fixture(scope="session")
+def engine_uri():
+    uri = os.environ.get("LOVELACE_ENGINE_URI", "http://localhost:14714")
```

Comment:
Hmmm, so does this try to get the environment variable `LOVELACE_ENGINE_URI`, but if it's not set it defaults to `http://localhost:14714`?

Reply:
Exactly. If you don't set any env vars then it'll behave the same as before. But if you ran it on a different port for example or (more likely) you run the engine in a VM or another machine then you could set that env var. 

The bigger practical problem for me was the path to the solutions dir; I can't run the tests without changing that from the default. But I figure that it's better not to hard-code things like this so it's a bit more flexible for different setups.

Reply:
Yeah it's definitely a huge improvement!

Couldn't find `os.environ.get` in the Python docs but just noticed now that `os.environ` is a dictionary and `dict.get()` allows you to specify a default. Good to know.